packages/keeper/src/process-jobs.js

```diff
@@ -29,14 +34,29 @@ const jobs = {
     }
 
     await queue.add({
-      name: JOB_UPDATE_BASE_URL,
+      name: JOB_PROCESS_BATCH_END,
```

Comment:
Instead of executing this job once, we create a job which creates a cronjob for 1 hour, which will periodically check for the baseURI and update it if necessary.

FYI, we don't do this check indefinitely because if theres a bug we would be updating the URI too much and run out of ETH on the account.